src/BridgedRoom.ts

```diff
@@ -176,6 +176,34 @@ export class BridgedRoom {
         this.puppetOwner = opts.puppet_owner;
 
         this.dirty = true;
+
+        // This almost certainly isn't the best place for this
+        this.joinAllSlackUsers();
```

Comment:
I would call `joinAllSlackUsers` during startup (queued, obviously), and in `actionLink`. Calling it in the constructor not only means we are not awaiting it, but also will cause unintended side effects by doing this in a constructor (e.g. if we do our cached rooms thing we talked about in #265)